install.sh

```
@@ -126,21 +151,82 @@ confirm_prompt() {
+    echo ""
+    echo -e "  ${WHITE}Options:${NC}"
+    echo -e "    ${CYAN}1.${NC} Stash changes (save for later)"
+    echo -e "    ${CYAN}2.${NC} Discard changes (permanent)"
+    echo -e "    ${CYAN}0.${NC} Cancel"
+    echo ""
+    read -rp "  Select option [0-2]: " change_option
+
+    case "$change_option" in
+        1)
+            print_status "Stashing changes..."
+            local stash_msg="Auto-stash $context on $(date '+%Y-%m-%d %H:%M')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)